interview-process/interview-recording.md

```diff
@@ -0,0 +1,46 @@
+# Recording Yourself
+
+## Time Schedule
+
+- 20 mins first partner recorded interview
+- 20 mins second partner recorded interview
+- 15 mins to watch your own recording
+- 15 mins debrief with your partner about what you observed
+- 5 mins self-assessment/reflection survey
+
+## Recording
+
+Use [Zoom](https://zoom.us) on your own computer to record your interview. You do not need to share this video with anyone, and yes, you can delete it right after if you want.
+
+## Questions
+
+### Partner 1 (choose to ask one of the two)
+
+View markdown when you are ready to ask
+<!--
+1. Find the 10 most popular words (popularity is determined by how often they occur) in a text file (or long string of text).
+2. Given any integer, print an English phrase that describes the integer (e.g. One Thousand Two Hundred Thirty Four).
+-->
+
+### Partner 2 (choose to ask one of the two)
+
+View markdown when you are ready to ask
+<!--
+3. Given two strings, write a method to decide if one is a permutation of the other (different ordering of the same letters)?
+4. Given two lists of unordered numbers, find the intersection of the two lists (the numbers that are in common).
+-->
+
+## Examples of things you are looking for in your recorded playback
+
+- Is your writing legible?
+- Do you block the write board when you talk?
+- Do you have long pauses? Or are they shorter in playback than they are in your head?
+- Do you appear confident?
+- How is your eye contact and demeanor?
+- Do you mumble?
+- Do you talk to the whiteboard or the interviewer?
+- Are you unnecessarily apologizing?
+- Are you saying things like "I don't know where this is going" instead of something like "I might rethink this later, but I'll start here"?
```

Comment:
```suggestion
- Are you saying things like, "I don't know where this is going" instead of something like, "I might rethink this later, but I'll start here"?
```